frontend/src/components/Helper/AsyncCalls.ts

```diff
@@ -29,3 +30,30 @@ export async function querySuggestions(dispatch: Function, query: string) {
 
     dispatch({type: ActionTypes.putSuggestions, suggestions: [/*retrieve suggestions from backend*/]})
 }
+
+/*
+* store selected items in local storage
+*/
+export async function storeItems(items: Item[]) {
+    try{
+        const jsonItems = JSON.stringify(items);
+        await AsyncStorage.setItem('@save_list', jsonItems);
+    } catch (e) {
+        //TODO
+        //could alert user
+    }
+}
+
+/*
+* store selected items in local storage
+*/
+export async function retrieveItems() {
+    try{
+        const jsonItems = await AsyncStorage.getItem('@save_list');
+        const items:Item[] = jsonItems != null ? JSON.parse(jsonItems) : null;
```

Comment:
Do you really need the ternary here? I'm pretty sure `JSON.parse(null)` would return `null` anyway

Reply:
Yea you are right

Reply:
If I just use `JSON.parse(jsonItems)` it gives an error saying "Argument of type 'string | null' is not assignable to parameter of type 'string'". I think 'JSON.parse()` accepts null strings but not the null value. Let me know if there is still a better approach than a ternary.